Xamarin.Forms.Core/SwipeItems.cs

```diff
@@ -7,12 +8,18 @@ namespace Xamarin.Forms
 {
 	public class SwipeItems : Element, IList<ISwipeItem>, INotifyCollectionChanged
 	{
-		readonly ObservableCollection<ISwipeItem> _internal;
+		readonly ObservableCollection<ISwipeItem> _swipeItems;
 
 		public SwipeItems()
 		{
-			_internal = new ObservableCollection<ISwipeItem>();
-			_internal.CollectionChanged += OnSwipeItemsChanged;
+			_swipeItems = new ObservableCollection<ISwipeItem>();
+			_swipeItems.CollectionChanged += OnSwipeItemsChanged;
+		}
+
+		public SwipeItems(IEnumerable<ISwipeItem> swipeItems)
```

Comment:
From @StephaneDelcroix 's comment
> constructors should probably be chained

would it make since to change `public SwipeItems()`
to

```C#
public SwipeItems() : this(Enumerable.Empty<ISwipeItem>())
```

Reply:
it does

Reply:
Changed!